airflow/cli/cli_parser.py

```diff
@@ -1104,6 +1113,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.connection_command.connections_delete'),
         args=(ARG_CONN_ID,),
     ),
+    ActionCommand(
+        name='export',
+        help='Export all connections',
+        func=lazy_load_command('airflow.cli.commands.connection_command.connections_export'),
```

Comment:
Can you add examples of calls for this command in the command description (description parameter not help parameter)?

Reply:
The help parameter appears in the task list, but the description parameter is available for help. For example: see `airflow dags show --help`.

Reply:
Got it.